io/zenoh-transport/src/unicast/link.rs

```diff
@@ -33,6 +37,27 @@ use zenoh_protocol::transport::TransportMessage;
 use zenoh_result::{bail, zerror, ZResult};
 use zenoh_sync::{RecyclingObjectPool, Signal};
 
+#[cfg(all(feature = "unstable", feature = "transport_compression"))]
+const HEADER_BYTES_SIZE: usize = 2;
+
+#[cfg(all(feature = "unstable", feature = "transport_compression"))]
+const COMPRESSION_BYTE_INDEX_STREAMED: usize = 2;
+
+#[cfg(all(feature = "unstable", feature = "transport_compression"))]
+const COMPRESSION_BYTE_INDEX: usize = 0;
+
+#[cfg(all(feature = "unstable", feature = "transport_compression"))]
+const COMPRESSION_ENABLED: u8 = 1_u8;
+
+#[cfg(all(feature = "unstable", feature = "transport_compression"))]
+const COMPRESSION_DISABLED: u8 = 0_u8;
+
+#[cfg(all(feature = "unstable", feature = "transport_compression"))]
+const BATCH_PAYLOAD_START_INDEX: usize = 1;
+
+#[cfg(all(feature = "unstable", feature = "transport_compression"))]
+const MAX_BATCH_SIZE: usize = u16::MAX as usize;
```

Comment:
I'm commiting a mistake here by setting this constant to u16::max? Since I believe the mtu is something to be set in the configuration and that is also negotiated between edges right?

Reply:
The maximum batch size is `u16::MAX`. I.e., the maximum amount of useful bytes to be sent excluding the header.